manim/plugins/__init__.py

```diff
@@ -1,3 +1,16 @@
 from __future__ import annotations
 
-from .import_plugins import *
+from .. import config, logger
+from .plugins_flags import get_plugins, list_plugins
```

Comment:
Same as above

Reply:
As said in my previous comment, relative imports are beneficial to some extent (i.e. if not going up too much). It is easier to refactor (if you rename a parent module, you'd have to rename your imports as well if you had used absolute imports).